servers/physics_server_2d.cpp

```diff
@@ -880,6 +880,23 @@ void PhysicsServer2D::_bind_methods() {
 
 PhysicsServer2D::PhysicsServer2D() {
 	singleton = this;
+
+	// World2D physics space
+	GLOBAL_DEF_BASIC("physics/2d/default_gravity", 980.0);
+	GLOBAL_DEF_BASIC("physics/2d/default_gravity_vector", Vector2(0, 1));
+	GLOBAL_DEF(PropertyInfo(Variant::FLOAT, "physics/2d/default_linear_damp", PROPERTY_HINT_RANGE, "-1,100,0.001,or_greater"), 0.1);
+	GLOBAL_DEF(PropertyInfo(Variant::FLOAT, "physics/2d/default_angular_damp", PROPERTY_HINT_RANGE, "-1,100,0.001,or_greater"), 1.0);
+
+	// GodotSpace2D
```

Comment:
```suggestion
	// PhysicsServer2D
```

The physics server implementation (and hence the implementation of "space") can be different from the default (via a module or GDExtension), but any implementation can still use the following global definitions.